### PR TITLE
Update wiki links in README

### PR DESCRIPTION
**Pull request by FantasqueX**
We can avoid redirecting :)

### CODE REVIEW

**Comment by mzz2017** [comment]
Please correct the word wiki to docs

**Comment by FantasqueX** [comment]
I have changed commit message to "Update document links in README"

**Comment by FantasqueX** [comment]
Actually, I don't understand what you want me to correct. Original word in README is wiki. Do you want me to correct that and change it to docs?

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ v2rayA mainly provides the following methods of installation:
 2. Docker
 3. Binary file and installation package
 
-See [**v2rayA - Wiki**](https://github.com/v2rayA/v2rayA/wiki/Usage)
+See [**v2rayA - Wiki**](https://v2raya.org/en/docs/prologue/introduction/)
```
Comment (mzz2017) [inline README.md:22]: Can you modify the title to v2rayA - Docs?

**Review by mzz2017 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (FantasqueX) [reply README.md]: Fixed

**Comment by mzz2017** [comment]
Good catch

**Pull request merged**